test/metabase/models/permissions_group_membership_test.clj

```diff
@@ -24,4 +24,16 @@
     (mt/with-temp User [user {:is_superuser true}]
       (db/delete! PermissionsGroupMembership :user_id (u/the-id user), :group_id (u/the-id (perms-group/admin)))
       (is (= false
-             (db/select-one-field :is_superuser User :id (u/the-id user)))))))
+             (db/select-one-field :is_superuser User :id (u/the-id user))))))
+
+  (testing "it should not let you remove the last admin"
+    (mt/with-single-admin-user [{id :id}]
+      (is (thrown? Exception
+                   (db/delete! PermissionsGroupMembership :user_id id, :group_id (u/the-id (perms-group/admin)))))))
```

Comment:
This tests existing behavior that simply wasn't tested before (probably since `with-single-admin-user` didn't exist).